{"name": "test_update_user", "status": "failed", "statusDetails": {"message": "AssertionError: 配置文件 config/config.yaml 读取失败，请检查路径或内容\nassert None is not None", "trace": "@pytest.fixture(scope=\"session\", autouse=True)\n    def init_token():\n        \"\"\"初始化环境配置\"\"\"\n        cfg = read_yaml(\"config/config.yaml\")\n>       assert cfg is not None, \"配置文件 config/config.yaml 读取失败，请检查路径或内容\"\nE       AssertionError: 配置文件 config/config.yaml 读取失败，请检查路径或内容\nE       assert None is not None\n\nconftest.py:9: AssertionError"}, "description": "更新用户信息", "start": 1761027515584, "stop": 1761027515584, "uuid": "59f8a34c-9711-47c3-9883-b6fdcfba158f", "historyId": "f73ab9f1c16a49f75d94bf0fd5db6533", "testCaseId": "f73ab9f1c16a49f75d94bf0fd5db6533", "fullName": "testcases.test_user_update#test_update_user", "labels": [{"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "test_user_update"}, {"name": "host", "value": "DESKTOP-UGQEI6T"}, {"name": "thread", "value": "11844-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.test_user_update"}], "titlePath": ["testcases", "test_user_update.py"]}